python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md

```diff
@@ -2,6 +2,17 @@
 In this directory, you will find examples on how to load GGUF model into `bigdl-llm`. For illustration purposes, we utilize the [llama-2-7b-chat.Q4_0.gguf](https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/tree/main) as a reference LLaMA2 GGUF model.
 >Note: Only LLaMA2 family models are currently supported
 
+## Support Model Family
+model_family refers to the conf['general.architecture'] of the GGUF file
```

Comment:
You should list the actual models, not the `conf['general.architecture']` here

Reply:
I have list the actual verified models under this line. The model_family is shown that what model_family we support now refer to [here](https://github.com/intel-analytics/BigDL/pull/9685/files#diff-1e0cbab4590ce1ad4f996f05357e881f092dcfc6ec3245ac2cbb22652a1d716eR46)

Reply:
Then you should list mistral as mistral, not llama